### DISCUSSION TITLE
Which is the Window Tiling Manager you are using in the video 1 for consult-omni?

### DISCUSSION THREAD

**Discussion by ghost** [armindarvish/consult-omni#19] [Q&A]
Is it macOS?
How did you customize the top bar looking
Thanks in advance.

**Comment by armindarvish** [comment]
I am using:
 - window manager: [yabai](https://github.com/koekeishiya/yabai) with [JankyBorders](https://github.com/FelixKratz/JankyBorders) 
 - bar: [SketchyBar](https://github.com/FelixKratz/SketchyBar)

**Comment by ghost** [comment]
I am using [AeroSpace](https://github.com/nikitabobko/AeroSpace) because I read that yabai (optionally) requires disabling System Integrity Protection, and that is what makes me doubt about using yabai. Did you had to disable SIP?

Aerospace had been working OK for me until now. But I tried [yequake](https://github.com/alphapapa/yequake) with my config, and it does not seem to handle well the floating emacsclient buffer created for consult-omni.

Also, I know next macOS version (Sequoia) will feature a better built-in window manager (don't know for sure if calling it a Tiling Window manager or what), and how well the macOS system will let the existing software to manipulate the windows.

**Reply by armindarvish** [reply]
You only need to disable "System Integrity Protection" for a subset of features. Tiling windows works even without it.

As far as I understand, next version of MacOS will have built-in support for on-demand tiling like snap-to-grid) and can provide features like Rectangle but not automatic tiling like a tiling window manager. But we'll have to wait and see I guess!

**Reply by ghost** [reply]
Thanks for the reply @armindarvish

**Comment by bechampion** [comment]
Also amazing work on omni consult ,  but what's the theme you're using , especially in the org-mode src blocks?

**Comment by armindarvish** [comment]
@bechampion I don't really have a single theme. I use a modus theme (like modus-vivendi) as baseline and then on top of that, I have customized the faces I mostly use in my everyday work, and that has resulted in what you see in the video.

I am not sure what you mean by the theme in or-mode src blocks. The org-mode src blocks are syntax highlighted natively, meaning by the major-mode of the source type.

**Reply by bechampion** [reply]
Hey Armin ,  i mean  that guide line your source blocks have on the right ,  that caught my eye somehow

**Reply by armindarvish** [reply]
@bechampion That is by using these packages:
1- [org-modern](https://github.com/minad/org-modern)
2- [org-modern-indent](https://github.com/jdtsmith/org-modern-indent)
